app/components/FavoriteCard.js

```diff
@@ -32,11 +32,21 @@ export const FavoriteCard = ({ title, poster_path, overview, release_date, vote_
     handleDeleteFave({ title, poster_path, overview, release_date, vote_average, movie_id, user_id, handleDeleteFave, updateFavorites})
   }
 
+  const checkIfFavorited = (title) => {
```

Comment:
Example PR Review Note: This is repeated twice in this PR - could we break it out into it's own file and add a test for it?